content/post/2020-10-14-dv-signal.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Using Electronic Health Records to Develop Real-Time COVID-19 Indicators"
```

Comment:
Since we now have only health insurance claims data, and that is also what's listed in the first sentence below, do we want to change the title to "Using Medical Insurance Claims data to Develop..."?